scout/quickstart.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Docker Scout quickstart
+keywords: scout, supply chain, vulnerabilities, packages, cves, scan, analysis, analyze
+description: >
+  Learn how to get started with Docker Scout to analyze images and fix vulnerabilities
+redirect_from:
+  - /atomist/get-started/
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout analyzes image contents and generates a detailed report of packages
+and vulnerabilities that it detects. Docker Scout can provide you with
+suggestions for how to remediate issues discovered by image analysis.
+
+The guide takes a vulnerable application image and shows you how to use Docker
+Scout to identify and fix those vulnerabilities, compare changes over time,
+and share results with your team.
+
+You can use [this example project](https://github.com/docker/scout-demo-service)
+to follow along by cloning its repository:
+
+```shell
+git clone https://github.com/docker/scout-demo-service.git
+```
+
+Move into the directory:
+
+```shell
+cd scout-demo-service
+```
+
+Build the image using the intended repository name, tagging it as "v1":
+
+```shell
+docker build -t <org-name>/scout-demo:v1 .
+```
+
+Create and push the repository on Docker Hub, again using the intended
+repository name:
+
+```shell
+docker push <org-name>/scout-demo:v1
+```
+
+> **Important**
+>
+> Make sure you login to the Docker CLI or Docker Desktop before pushing.
+> {: .important}
+
+Docker Scout analyzes all local images by default. To analyze images in
+remote repositories, you need to enable it first.
+[You can do this from Docker Hub, the Docker Scout Dashboard, and CLI](/scout#).
+
+This guide uses the Docker Scout CLI [`repo enable`](/engine/reference/commandline/scout_repo_enable)
+command to enable analysis on an existing repository with the following command:
+
+```shell
+docker scout repo enable <org-name>/scout-demo
+```
+
+After building, using Docker Desktop or the `docker scout` CLI command, you
+should be able to see vulnerabilities detected by Docker Scout.
+
+Using Docker Desktop, select the image name in the **images** tab to see the image layer view.
+In the image hierarchy section, you can
+see which layers introduce vulnerabilities and the details of those.
+
+Focus on the vulnerability introduced in layer 5.
+
+Select layer 5, toggle the disclosure triangle next to **express 4.17.1**,
+and then the CVE ID (in this case, "CVE-2022-24999⁠") to see details of the
+vulnerability.
+
+![Screenshot showing highlighting a CVE in Docker Desktop](./images/scout-onboarding-dd-v1-cvve-focus.png)
```

Comment:
I get why there is a screenshot here, but this isn't actually doing what we want it to do. I think if anything, we could include a short screen capture showing these clicks. But I wouldn't keep this screenshot as-is